src/components/TeacherViews/Onboarding/Onboarding.view.tsx

```diff
@@ -98,96 +66,48 @@ export const OnboardingView = () => {
         }
       >
         <Logo height={48} width={104} variant="with text" />
-        <OakHeading tag="h2" $font="heading-light-5">
-          Select your school
+        <OakHeading
+          $mb={"space-between-s"}
+          $mt={"space-between-m"}
+          tag="h2"
+          $font="heading-light-6"
+        >
+          Do you work in a school?
         </OakHeading>
+        <OakBox $pv={"inner-padding-xl"}>
+          <OakRadioGroup
+            onChange={(value) =>
+              setWorksInSchool(value.target.value === "yes" ? true : false)
```

Comment:
ternary here is a little redundant.

```suggestion
              setWorksInSchool(value.target.value === "yes")
```